build/workbox.config.json

```diff
@@ -1,7 +1,7 @@
 {
   "globDirectory": "./",
   "globPatterns": [
-    "_gh_pages/**/*.{html,css,js,png,jpg}"
+    "_gh_pages/**/*.{html,css,js,json,png,jpg}"
```

Comment:
With respect to the favicon issue you're seeing, is it possible to view your current build in Chrome Canary and take a screenshot of what is in Application Panel -> Cache Storage? Favicon issues like this are often due to the asset either not existing in the final build, the globPattern not being sufficient (don't believe this is an issue here) or something else.

Reply:
@addyosmani: I see the favicons being in the workbox cache:

![cache](https://user-images.githubusercontent.com/349621/30454461-1868c56e-99a5-11e7-91c3-1684bdc46717.png)

And thanks for the reply :)